confluent_platform/tox.ini

```
@@ -0,0 +1,22 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py38
+envlist =
+    py{27,38}
+
+[testenv]
+description =
+    py38: e2e ready
+dd_check_style = true
```

Would this be relevant?

```suggestion
dd_check_style = true
dd_check_types = true
dd_mypy_args = --py2 tests/
```

---

I think for `JMX` integration, there is no much value.